packages/node-experimental/src/integrations/http.ts

```diff
@@ -210,3 +215,11 @@ function getHttpUrl(attributes: Attributes): string | undefined {
   const url = attributes[SemanticAttributes.HTTP_URL];
   return typeof url === 'string' ? url : undefined;
 }
+
+/**
+ * Checks whether given host points to Sentry server
+ */
+function isSentryHost(host: string | undefined): boolean {
```

Comment:
we already have something like this in a few places. Maybe we can extract into a function.

Reply:
good point, will move it as a util to `@sentry/core` (can't be in utils as it needs access to `getCurrentHub()`!)